substrate/frame/revive/src/lib.rs

```diff
@@ -83,9 +83,9 @@ use sp_runtime::{
 
 pub use crate::{
 	address::{create1, create2, AccountId32Mapper, AddressMapper},
-	debug::Tracing,
 	exec::{MomentOf, Origin},
 	pallet::*,
+	tracing::Tracer,
```

Comment:
Let's not re-export this type when the module is already public. I want to keep the top level rust doc clean.